sql-plugin/src/main/311until340-all/scala/com/nvidia/spark/rapids/shims/GpuSparkPath.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.shims
+
+import scala.language.implicitConversions
+
+/**
+ * A helper type for the implicit conversion between `String` and `SparkPath`.
+ */
+final case class GpuSparkPath(path: String)
+
+object GpuSparkPath {
+  implicit def GpuSparkPathToString(path: GpuSparkPath) = path.path
```

Comment:
```suggestion
  implicit def gpuSparkPathToString(path: GpuSparkPath) = path.path
```
Method name usually starts with lower char.

Reply:
Fixed. Thank you for catching it.